types/light-sdk/index.d.ts

```diff
@@ -3,637 +3,631 @@
-         * @deprecated
-         * @param params - 接口入参
-         * @param params.type -关闭的键盘类型（price:交易价格键盘 volume:交易量键盘 search:股票代码搜索键盘）
-         * @param cb
-         */
-        tradekeyboardHide(params: any, cb: (res: CallbackResultNoData) => void): void;
-        /**
-         * 通过js分享内容到各平台
-         * @param params - 接口入参
-         * @param params.title            分享标题
-         * @param params.content            分享的描述信息
-         * @param params.url            用户点击分享后打开的链接地址，
-         * @param params.image    一般是标准http或https请求,可支持格式
-         * 1.远程图片url，必须以http://或https://开头
-         * 2. 图片Base64编码，以base64://开头
-         * 3. gmu／gmu_icon目录下的本地文件，相对路径且不包括文件后缀，如使用图片gmu/gmu_icon/test.png, 则次此参数为test"
-         * @param params.type    可选值：
-         * “webpage”（分享网页)
-         * “image”（分享图片) (注意:分享图片目前支持微信，微博，qq，钉钉。其他则不支持分享图片功能）
-         * “file”（分享文件) (注意:本地地址需要为文件预览或文件下载接口所传的文件存放的相对路径地址。文件分享目前只支持微信，其他如qq，微博，钉钉则不支持文件分享功能。注意:微信朋友圈不支持文件分享功能）
-         * (默认值：”webpage”)
-         * @param params.channel 设置分享渠道，用户可指定将信息分享到哪个渠道上 ，注意：若配置了此参数，则无分享对话框弹出UI，不配置则会根据开发者的share.gmu中的配置生成动态生成分享对话框弹出，供用户选择分享渠道。
-         * 可选值：
-         * “weixin” （微信）
-         * “weixin_timeline”（微信朋友圈）
-         * “qq”（QQ）
-         * “weibo”(微博)
-         * “dingtalk”（钉钉）
-         * “system”(系统分享)”
-         * @param cb 功能处理后的回调函数
-         */
-        socialShare(params: any, cb: (res: CallbackResultNoData) => void): void;
-        /**
-         * 三方登录接口
-         * @param params - 接口入参
-         * @param params.type            登录平台类别,只支持qq,weibo,weixin三个type
-         * @param cb 功能处理后的回调函数
-         */
-        socialLogin(params: any, cb: (res: CallbackResult) => void): void;
-        /**
-         * 通过js判断各个平台app在设备上是否已安装
-         * @param params - 接口入参
-         * @param params.type -平台类型，目前type值只支持'qq'，'weibo'，'weixin'三个
-         * @param cb 功能处理后的回调函数
-         */
-        socialAppInstalled(params: any, cb: (res: CallbackResult) => void): void;
-        /**
-         * 苹果内购接口
-         * @param params - 接口入参
-         * @param params.apple_product_id -苹果商品ID(需要在苹果itunes connect平台创建商品)
-         * @param params.orderId -商户订单号(此订单号为客户自己业务生成的订单号，用于最后一步校验苹果收据时做检测唯一性使用，保证内购链路完整性)
-         * @param cb 功能处理后的回调函数
-         */
-        iapPurchase(params: any, cb: (res: CallbackResult) => void): void;
-        /**
-         * 通过js传入经服务器加签后的订单信息(orderInfo)并调用相应支付渠道SDK的支付接口
-         * @param params - 接口入参
-         * @param params.channel_id -021:支付宝钱包022:微信钱包
-         * @param params.orderInfo			- 具体内容见下方的”其他(当支付渠道为微信钱包时/当支付渠道为微信钱包时)”项
-         * @param params.isSandBox			-是否进入沙箱环境（只对支付宝有效）
-         * @param cb 功能处理后的回调函数
-         */
-        tradePay(params: any, cb: (res: CallbackResult) => void): void;
-    }
+interface CallbackResult {
+    info: Info;
+    data: any;
+}
 
-    function register(options: any): void;
-    const config: any;
-    const native: Native;
-    const net: any;
-    const openAPI: any;
+interface Native {
```

Comment:
Do you want to `export` these interfaces? They effectively were `export`ed in the namespace before.

Reply:
I modified it again, please check again! thanks~